config/install/system.site.yml

```diff
@@ -1,5 +1,6 @@
 langcode: en
 name: 'Convivial Demo'
+mail: contact@morpht.com
```

Comment:
Please revert these changes. The email ID should not be in the profile.

Reply:
Done